pages/index.tsx

```diff
@@ -144,77 +274,258 @@ export default function Home() {
+                badge={
+                  <span
+                    className={styles.badgeYellow}
+                    style={{ top: "-30px", left: "6%" }}
+                  >
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    {history?.[0]?.location ?? ""}
+                  </span>
+                }
+                badge2={
+                  <span
+                    className={styles.badgeYellow}
+                    style={{ top: "-30px", left: "35%" }}
+                  >
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    {history?.[0]?.location ?? ""}
+                  </span>
+                }
+                badgeVirginia={
+                  <span className={`${styles.badgeYellow} ${styles.virginia}`}>
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    n.virginia
+                  </span>
+                }
+              />
+            </div>
+            <div className={styles.cardInfo}>
+              <div className={styles.numbers}>
+                <h4>{num.format((1_000 / cacheLatency) * 60)}</h4>
+                <span>queries per minute</span>
+              </div>
+              <div className={styles.numbers}>
+                <h4>{ms.format(cacheLatency)}</h4>
+                <span>latency</span>
+              </div>
+              <p>
+                The result of the database query is cached at the Accelerate
+                caching node in&nbsp;
+                <span className={styles.badgeYellow}>
+                  <img
+                    src="/locationDot.svg"
+                    color={defaultTheme.colors.orange[400]}
+                    width="8px"
+                    height="15px"
+                  />
+                  {history?.[0]?.location ?? ""}
+                </span>
+                &nbsp; and retrieved from there:
+              </p>
+              <ul>
+                <li>✨ Reduced latency</li>
+                <li>🚀 Increased Query Capacity</li>
+                <li>🌟 Optimal Resource Utilization</li>
+              </ul>
+              <div
+                className={`${styles.expandBar} ${styles.with} ${
+                  showWith && styles.active
+                }`}
+                onClick={() => toggleWith(!showWith)}
+              >
+                Expand<span className={styles.mobile}> to view</span> Prisma
+                Client query
```

Comment:
```suggestion
                Expand{' '}<span className={styles.mobile}>to view</span>{' '}Prisma Client query
```
The space is not working correctly here 🤔 
<img width="251" alt="image" src="https://github.com/prisma/accelerate-speed-test/assets/8246284/eb7175a9-f104-4a09-bab0-6d03b650893b">